### DISCUSSION TITLE
Override JavaScript methods

### DISCUSSION THREAD

**Discussion by cchidhu** [aws/aws-cdk#27640] [General]
I have the following TypeScript class where one of the methods needs to be overridden in the implementation. I am ensuring this in the constructor.
<img width="1060" alt="image" src="https://github.com/aws/aws-cdk/assets/1626664/f7cbfdce-4a22-43c9-a3bb-20b8696ccc24">

Implementation of the above class in Python,
<img width="892" alt="image" src="https://github.com/aws/aws-cdk/assets/1626664/2f114769-ad2f-416c-8e6f-5433a9fdd81b">

The foreign application implementing the above class is experiencing unexpected results even after the 'dummy_green_method' is overridden in the implementation. I expect the super constructor not to throw an error in the implementation because the method has been overridden. However, this is not the case. Please provide guidance on what I might be missing here.

**Comment by indrora** [comment]
I don't believe we currently support fully cross-language overrides. This is more appropriate for the aws/jsii ticket tracker.
